lib/ibandit/iban_assembler.rb

```diff
@@ -39,10 +46,7 @@ def self.assemble_at_bban(opts)
       #
       # Padding:
       #   Add leading zeros to account number if < 11 digits.
-      [
-        opts[:bank_code],
-        opts[:account_number].rjust(11, '0')
-      ].join
+      [opts[:bank_code], opts[:account_number]].join
```

Comment:
Almost all of these methods are just a join. Should we do that by default and only hadle exceptions (Italy) separately?